### ISSUE TITLE
Java docs markup is not showing up

### ISSUE THREAD

**Issue by qvanphong** [ArkEcosystem/docs#92]
**Expected Behavior**
In maven dependency import, we should see `<markup>`.

**Current Behavior**
Markup did not show up.

**Possible Solution**

Since I tried to clone the repo https://github.com/ArkEcosystem/docs/ but I have no idea how to make it run on my local.
The problem should inside https://github.com/ArkEcosystem/docs/blob/master/docs/sdk/java/crypto/intro.md.blade.php 

**Steps to Reproduce (for bugs)**

👉 Go to https://ark.dev/docs/sdk/java/crypto/intro

**Comment by sleepdefic1t** [comment]
Sorry for the late reply, @qvanphong !

We've found the issue and will be implementing a fix soon.

We'll keep this issue open in the mean time 👍 

Thank you!

**Comment by qvanphong** [comment]
Thanks @sleepdefic1t for noticed my submitted issue :D

**Comment by qvanphong** [comment]
Hey @sleepdefic1t, I see another problem on 
https://ark.dev/docs/sdk/java/crypto/intro

The address for fork ``git clone https://github.com//java-crypto`` isn't correct, it should be ``https://github.com/ArkEcosystem/java-crypto.git``.

**Comment by sleepdefic1t** [comment]
@qvanphong 
No problem, and Yeah.. this is a backend issue affecting all docs.

tl;dr
Certain characters are not being correctly parsed from markdown into html. The fix is in-progress and should be ready soon 👍

```
git clone https://github.com/<githubusername>/java-crypto
```
> https://github.com/ArkEcosystem/docs/tree/master/docs/sdk/java/crypto/getting-started.md.blade.php#L58

Thanks again for bringing this to our attention 🎉

**Comment by sleepdefic1t** [comment]
@qvanphong 
Just circling back to let you know this was recently resolved 👍 

![](https://i.ibb.co/ZfWq9gb/4ejr74n0iabe4ttflo78.png)
> [https://ark.dev/docs/sdk/java/crypto/intro](https://ark.dev/docs/sdk/java/crypto/intro)

---

I'll go ahead and close this issue out.
Feel free to let us know if you have any other questions or concerns relating to this.

Thank you again!
